### DISCUSSION TITLE
broken textures

### DISCUSSION THREAD

**Discussion by StrajZoni** [pudingus/Max4dsTools#8] [Q&A]
I tried changing models through 3d max 9, but the textures in the game on the new models broke. Does this mean that I should download a new version of the program?

**Comment by pudingus** [comment]
3ds Max 9 is not supported, get something newer, 2014 atleast. The versions 2009-2013 are technically supported too, but explicit normals don't work there. So get 2014+, ideally 2020+, but not 2024 because it is not supported yet.

**Reply by StrajZoni** [reply]
OK, thanks)
